Neuron/Sections/Guide/GuideViewController.swift

```diff
@@ -11,24 +11,22 @@ import UIKit
 private var GuideOnceTokenAssiciationKey = 0
 
 class GuideViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
-    struct GuideItem {
-        let title: String
-        let subTitle: String
-        let imageName: String
-    }
-
     private let items = [
-        GuideItem(title: "甄选全球最新DAPP", subTitle: "一应俱全", imageName: "guide_1"),
-        GuideItem(title: "多重安全算法保护", subTitle: "钱包秘钥惟您掌握", imageName: "guide_2"),
-        GuideItem(title: "开启区块链之旅", subTitle: "探索无限可能", imageName: "guide_3")
+        "Guide.imageName.1".localized(),
```

Comment:
@cezres An approach for image localization is to move the images out of asset catalog (meaning putting them in Resources group), and localize them directly. I believe that's the correct and better way to do so, although we won't be able to use asset catalog for this which is OK.